engine/src/gldrv/winsys.cpp

```diff
@@ -493,7 +516,10 @@ void winsys_shutdown() {
 void winsys_show_cursor(bool visible) {
     static bool vis = true;
     if (visible != vis) {
-        SDL_ShowCursor(visible);
+        if (visible)
+            SDL_ShowCursor();
+        else
+            SDL_HideCursor();
```

Comment:
our coding style requires using braces around all code blocks so this needs to be:
```c++
if (visible) {
    SDL_ShowCursor();
else {
    SDL_HideCursor();
}
```